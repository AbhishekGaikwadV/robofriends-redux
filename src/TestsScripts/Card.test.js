import { shallow, mount, render } from 'enzyme';
import React from 'react';
import Card from '../components/Card';

// npm test -- --coverage


describe("Card", ()=>{
    it("Card SnapShot", () => {
        expect(shallow(<Card />)).toMatchSnapshot();
    });
} );



