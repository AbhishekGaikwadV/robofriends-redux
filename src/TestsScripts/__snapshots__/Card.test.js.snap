// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Card SnapShot 1`] = `ShallowWrapper {}`;

exports[`Card Name 1`] = `ShallowWrapper {}`;
